include/mxnet/swap.h

```diff
@@ -24,15 +28,20 @@ class Swap {
   static std::shared_ptr<Swap> _GetSharedRef();
   void SwapOut(unsigned required_memory, int device);
   void SwapIn(SwapInfo *info);
-  void* GetAddr(handle_id_t handle_id, size_t size);
   void SetAddr(handle_id_t handle_id, void* dptr, size_t size, int dev_id);
-  void DelAddr(handle_id_t handle_id, size_t size);
+  void DelAddr(handle_id_t handle_id);
+  void* GetAddr(handle_id_t handle_id);
+  int UpdateFree(int device); 
```

Comment:
You may want to add a comment since this function name is less obvious than other function names.